Checkout/Checkout/Source/Validation/Validators/CardNumberValidator.swift

```diff
@@ -32,9 +33,32 @@ final class CardNumberValidator: CardNumberValidating {
       return .success(cardScheme)
     case .none:
 
-      let cardScheme = cardTypeMatch(cardNumber, \.eagerCardNumberRegex) ?? .unknown
-      return .success(cardScheme)
+      return .success(.unknown)
```

Comment:
I am wondering, can validation be successful with `unkown` schema?

Reply:
Yeah I think this needs to be discussed again with the team. `.unknown` is returned when a string of digits has no regex match, which is confusing for merchants. This should be an error